### ISSUE TITLE
Not an AI agent tool

### ISSUE THREAD

**Issue by ihor-sokoliuk** [sammcj/n8n-nodes-searxng#5]
n8n can't use this node as an AI Agent tool.

`    // Add AI tool metadata` in `nodes/Searxng/Searxng.node.ts` did not help.

I could try adding by myself and contributing to this repo if you share how you test your code. Documentation is obsolete, and commands from there do not work for me.

**Comment by ihor-sokoliuk** [comment]
@sammcj any updates on this matter?

**Comment by sammcj** [comment]
Hey, sorry I'm not really using N8N at the moment, if I come back to it in the new year I might have a look at this.

**Comment by ihor-sokoliuk** [comment]
@sammcj Not a problem.

Please share the commands you use for your node development, if you recall any. Development instructions from the n8n official doc do not work.

**Comment by sammcj** [comment]
What do you mean "the commands"? I just wrote the code, then tested it locally as per https://docs.n8n.io/integrations/creating-nodes/test/run-node-locally/, then published to npmjs.com and installed via the UI.

**Comment by ihor-sokoliuk** [comment]
Yes, the commands from the official doc are not working now since n8n switched to pnpm.

I am not a NodeJS dev, but I spent a while finding the commands that work for me.
I run them to apply code changes at my local n8n server.

```shell
# in the node folder
rm -rf dist/ node_modules/
npm install n8n
npm run build
pnpm link .

# in the ~/.n8n/custom folder
pnpm unlink /workspaces/n8n-nodes-searxng
pnpm link /workspaces/n8n-nodes-searxng
n8n start
``` 

I am about to create a PR with new feature.

Happy New Year @sammcj !!!
